spec/openhab/dsl/types/decimal_type_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe 'OpenHAB::DSL::Types::DecimalType' do
+  it 'is inspectable' do
+    expect(DecimalType.new(10)).to eq '10'
```

Comment:
missing `.inspect` here?

Reply:
yup, whoops. TIL `DecimalType.new(10) == '10'` is true. fixed, and also changed to `eql` matcher to be more particular in the future.